### DISCUSSION TITLE
Async integration to paste island

### DISCUSSION THREAD

**Discussion by AxDevv** [BG-Software-LLC/SuperiorSkyblock2#1357] [Denied Feature Requests] [answered]
Hello,

I encounter TPS drops when creating an island due to the paste of the schematic, I would like to propose an Asynchronous Mode with FAWE API that would avoid the TPS drops.

**Comment by OmerBenGera** [comment, marked as answer]
Its not possible to place blocks asynchronous in Minecraft - neither FAWE do that.

SSB's way of pasting schematics is way faster than any other. I recommend sticking with it and if you believe there is are tps drops, open an issue and include there a spark profiler
